### ISSUE TITLE
Crash with `btrbk list all` when fresh install (no shapshots or backups yet)

### ISSUE THREAD

**Issue by sep2** [digint/btrbk#367]
The system is Archlinux latest.

```
> doas btrbk list all

ERROR: process died unexpectedly (btrbk v0.31.0)
Please contact the author: Axel Burri <axel@tty0.ch>

Stack Trace:
----------------------------------------
Use of uninitialized value in addition (+) at /usr/bin/btrbk line 6797.
 at /usr/bin/btrbk line 346.
        main::__ANON__("Use of uninitialized value in addition (+) at /usr/bin/btrbk "...) called at /usr/bin/btrbk line 6797

```

**Comment by digint** [comment]
Thanks for reporting!

Fixed in: 8d25852ce23bffd38ee698cb6ef03b20c49e00b7

**Comment by digint** [comment]
Included in btrbk-0.31.1
